chef-config/lib/chef-config/path_helper.rb

```diff
@@ -149,6 +149,13 @@ def self.escape_glob(*parts)
       path.gsub(/[\\\{\}\[\]\*\?]/) { |x| "\\"+x }
     end
 
+	# This function does not switch to backslashes for windows
+	# This is because only forwardslashes should be used with dir (even for windows)
```

Comment:
need to knock off 2 leading spaces here.

more importantly it would be good to note that escape_glob was deprecated with a comment above that function, i think it is always wrong now.